### ISSUE TITLE
Share current price

### ISSUE THREAD

**Issue by dllud** [drpout/boilr#248]
I often feel the need to share the current price of some currency via Instant Messaging or Email. There should be a button for that and also some way to copy the price to the clipboard.

I was thinking about transforming the current price box into a button, like we already did with the upper and lower limits. That button would show a popup featuring buttons to share or copy the current price. What do you think about it?

**Comment by algazarra** [comment]
I agree, but would just get the function of copying to the clipboard. Also, this will require some design mods so that it becomes clear that clicking A gets you to the alarm's settings, clicking B copies current price, and so on.

**Comment by dllud** [comment]
> would just get the function of copying to the clipboard

That way we don't even need a popup. Clicking the price would copy it to the clipboard and show a toast saying "Current price copied to the clipboard.". 
Though, I've seen users that don't know that Android has a clipboard. Everyone is used to the share buttons.

IMO this particular addition requires no design changes. Things will behave consistently, it's clear what all clickable areas do:
- Clicking the left-side area, where there is the pair and exchange, takes you to the alarm settings, where the first thing you can do is to change those two parameters.
- Clicking the upper/lower limits or change value allows you to change them.
- Clicking the power button turns the alarm on and off.
- Clicking the current price does not allow you to change it since it's something that cannot be changed (it comes from the exchange). But it allows you to do something with the price (copy to clipboard).
